crates/pipeline_manager/src/db/test.rs

```diff
@@ -524,10 +572,11 @@ fn db_impl_behaves_like_model() {
     // ended which meant the DB would not shut-down after this test.
     let mut config = Config::default();
     config.max_shrink_iters = u32::MAX;
+    config.source_file = Some("src/db/test.rs");
```

Comment:
Is this safe when running tests from vscode and from the commandline from different directories (e.g. cargo test from crates/pipeline_manager vs from project root)?

Reply:
Just hit a bug with running in vscode and it was updating the correct files (running from inside vscode)